pallets/automation-time/src/lib.rs

```diff
@@ -406,54 +439,54 @@ pub mod pallet {
 		/// Schedule task success.
 		TaskScheduled {
 			who: AccountOf<T>,
-			task_id: T::Hash,
+			task_id: TaskId<T>,
 		},
 		// Cancelled a task.
 		TaskCancelled {
 			who: AccountOf<T>,
-			task_id: T::Hash,
+			task_id: TaskId<T>,
 		},
 		/// Notify event for the task.
 		Notify {
 			message: Vec<u8>,
 		},
 		/// A Task was not found.
 		TaskNotFound {
-			task_id: T::Hash,
+			task_id: TaskId<T>,
```

Comment:
Should we be including the `account_id` as part of all the success and failure events since tasks can no longer be identified with just their task id?

Reply:
hmmm I should probably change all the events who send `task_id` to also send `who`